tests/end_to_end.test.js

```diff
@@ -1242,5 +1242,21 @@ const Pulsar = require('../index.js');
       await consumer.close();
       await client.close();
     });
+
+    test('Client/getPartitionsForTopic', async () => {
+      const client = new Pulsar.Client({
+        serviceUrl: 'pulsar://localhost:6650',
+        operationTimeoutSeconds: 30,
+      });
+
+      const topic = 'persistent://public/default/get-partitions-for-topic';
+      await client.createProducer({
```

Comment:
You can use admin [REST API](https://pulsar.apache.org/admin-rest-api/?version=3.0.0#) to create specified partitions topic, and then to assert `getPartitionsForTopic`.